packages/gg_images/PKGBUILD

```
@@ -0,0 +1,37 @@
+pkgname=gg_images
+pkgver=20.fe35153
+pkgrel=1
+pkgdesc='GG profile photo downloader'
+arch=('x86_64')
+groups=('blackarch')
+url='https://codeberg.org/nanoory/GG_Images'
+license=('GPL-3.0')
```

```suggestion
license=('GPL3')
```

Use license names as found in `/usr/share/licenses/common/` wherever applicable